core/src/main/java/gov/nist/secauto/metaschema/core/model/constraint/ConstraintValidationFinding.java

```diff
@@ -292,8 +292,8 @@ public ConstraintValidationFinding build() {
 
       List<? extends INodeItem> subjects = this.subjects == null ? CollectionUtil.emptyList() : this.subjects;
 
-      assert subjects != null;
-      assert kind != null : "Kind must not be null";
+      assert subjects != null : "subjects must not be null";
+      assert kind != null : "kind must not be null";
```

Comment:
Non-blocking: just wanted to be clear from reading the code (sorry I have not stepped through a debugger with code on this one yet): `kind` is the type of constraint (`expect`, `matches`, `allowed-values`) just not using the name `type` to reduce confusing variable names?

Reply:
[Kind](https://github.com/metaschema-framework/metaschema-java/blob/928d8b4bde2ac8abc1981cdc14b158f3f66fc738/core/src/main/java/gov/nist/secauto/metaschema/core/model/validation/IValidationFinding.java#L24) is the type of finding, i.e. pass, fail, informational.

Reply:
Oh ok, I get what you mean. I had to re-type this comment three times as I confusedly wondered what the difference between kind and severity are, but then I realized informational can mean something for both properties a validation finding but mean very different things, apologies!